src/OpenIDConnectClient.php

```diff
@@ -640,6 +651,21 @@ private function requestAuthorization() {
             $auth_params = array_merge($auth_params, array('response_type' => implode(' ', $this->responseTypes)));
         }
 
+        // If the client supports Proof Key for Code Exchange (PKCE)
+        if (!empty($this->getCodeChallengeMethod()) && in_array($this->getCodeChallengeMethod(), $this->getProviderConfigValue('code_challenge_methods_supported'))) {
+            $codeVerifier = bin2hex(random_bytes(64));
```

Comment:
The limitation to run the PKCE feature, is that requires PHP 7 because of [random_bytes()](https://www.php.net/manual/en/function.random-bytes.php) function.
If you can suggest an alternative way for this, please let me know.

Reply:
@nikosev Thanks for the PR! We could maintain compatibility by using a polyfill. (https://github.com/paragonie/random_compat) Thoughts?

Reply:
@jumbojett I tested this polyfill with PHP 5.6.40 and random_compat 2.0.19 and worked fine.
I have pushed a commit with the addition of this library to composer.json